src/inbox/mod_inbox_rdbms.erl

```diff
@@ -146,6 +169,12 @@ clear_inbox(Username, Server) ->
     Res = clear_inbox_rdbms(LUsername, LServer),
     check_result(Res).
 
+-spec clear_inbox(Server :: binary()) -> ok.
```

Comment:
`check_result/1` might return `ok` or `{ok, Val}`. Are we positive that in this function it's called in such a way that will always result in `ok`? If not, this spec is incorrect.